src/panels/config/entity_registry/ha-config-entity-registry.ts

```diff
@@ -45,9 +45,62 @@ class HaConfigEntityRegistry extends LitElement {
 
   private _filteredEntities = memoize(
     (entities: EntityRegistryEntry[], showDisabled: boolean) =>
-      showDisabled
+      (showDisabled
         ? entities
         : entities.filter((entity) => !Boolean(entity.disabled_by))
+      ).map((entry) => {
+        const state = this.hass!.states[entry.entity_id];
+        return {
+          ...entry,
+          icon: state
+            ? stateIcon(state)
+            : domainIcon(computeDomain(entry.entity_id)),
+          name:
+            computeEntityRegistryName(this.hass!, entry) ||
+            this.hass!.localize("state.default.unavailable"),
+          enabled: !entry.disabled_by,
```

Comment:
Since we have a custom template renderer for this field, we might as well not pre-process it here